File: app/src/main/java/com/footinit/baseavd/utils/NetworkUtils.java
package com.footinit.baseavd.utils;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

/**
 * Created by Abhijit on 08-11-2017.
 */

public final class NetworkUtils {


    private NetworkUtils() {
        //PC
    }

    public static boolean isNetworkConnected(Context context) {
        ConnectivityManager cm = (ConnectivityManager)
                context.getSystemService(Context.CONNECTIVITY_SERVICE);
        NetworkInfo networkInfo = cm.getActiveNetworkInfo();

        return networkInfo != null && networkInfo.isConnectedOrConnecting();
    }
}

File: app/src/main/java/com/footinit/baseavd/utils/NetworkUtils.kt
package com.footinit.baseavd.utils

import android.content.Context
import android.net.ConnectivityManager
import android.net.NetworkInfo

/**
 * Created by Abhijit on 08-11-2017.
 */

object NetworkUtils {

    fun isNetworkConnected(context: Context): Boolean {
        val cm = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        val networkInfo = cm.activeNetworkInfo

        return networkInfo != null && networkInfo.isConnectedOrConnecting
    }
}//PC
